### ISSUE TITLE
How to work with entity relations?

### ISSUE THREAD

**Issue by ishener** [terence410/ts-datastore-orm#6]
I couldn't find any example for referencing an entity from another entity. Are relationships supported? Things like One-To-Many, Many-To-Many etc.

If not, is this something that is planned?

**Comment by terence410** [comment]
Yes, you may say it's planned. I just follow the design the datastore and make it to be used with ease.

You may take a look the ancestor usage in this test: https://github.com/terence410/ts-datastore-orm/blob/master/tests/general.test.ts

Datastore utilize a concept of "key" and "ancestor", many things are inter related inside, which in a result make things harder to integrate with the traditional type of "relationship". If you further play with transaction, those ancestor become even more critical if u want some strong atomic feature during entity reading.
